src/pages/invitee/EventInvite.jsx

```diff
@@ -47,18 +76,23 @@ const EventInvite = () => {
     }, 2000)
   }
 
+  useEffect(() => {
+    getEventDetails();
+  }, []);
+
   return (
     <div>
       <Navbar />
       <div className="mt-28 mb-10 mx-auto md:mx-32">
         <div className="mx-2 md:mx-0 text-center w-full">
           <h1 className="text-2xl font-bold md:text-3xl">Hello, there.</h1>
           <p className="leading-6 text-gray-600 font-sm">
-            You have been invited to dinner by{" "}
-            <span className="text-blue-700 font-bold"> Mathew Mathais.</span>
+            You have been invited to {eventData ? eventData.data.event_type : ""} at
+            <span className="text-blue-700 font-bold"> {eventData ? eventData.data.location : ""}</span>
             <br /> You can view the details below..
           </p>
           {declinedInvite ? <p className="font-bold text-red-900">You have succesfully declined this invite... Redirecting to your homepage soon</p> : null}
+          {resultMsg ? <p className="text-red-500">{resultMsg}</p> : ""}
         </div>
         <div className="my-8 border py-5 rounded-lg flex justify-center items-center ">
           <form onSubmit={addParticipant}>
```

Comment:
- [ ] Kindly add validations to all your email such that when a user enters a wrong or invalid email, an error message should popup so as to avoid wrong data being sent to the backend